packages/mobile/src/identity/privateHashing.ts

```diff
@@ -70,6 +72,9 @@ function* doFetchPhoneHashPrivate(e164Number: string) {
   }
 
   Logger.debug(`${TAG}@fetchPrivatePhoneHash`, 'Salt was not cached, fetching')
+  if (!balanceSufficientForQuotaRetrieval()) {
```

Comment:
It would be better for this to live in the Contract kit ODIS utils but if we're in a hurry to get this merged feel free to just leave a TODO